sdk/keyvault/keyvault-secrets/package.json

```
@@ -102,7 +102,11 @@
   },
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
-    "@azure/core-http": "^2.0.0",
+    "@azure/core-client": "^1.5.0",
+    "@azure/core-http-compat": "^1.3.0",
+    "@azure/core-rest-pipeline": "^1.8.0",
+    "@azure/core-auth": "^1.3.0",
+    "@azure/core-util": "^1.0.0-beta.1",
```

I think you would want to depend on the GA version
```suggestion
    "@azure/core-util": "^1.0.0",
```

---

Thanks :)